lib/Server.js

```diff
@@ -889,23 +894,18 @@ class Server {
         // support "." as a subdomain wildcard
         // e.g. ".example.com" will allow "example.com", "www.example.com", "subdomain.example.com", etc
         if (allowedHost[0] === '.') {
-          // "example.com"
-          if (hostname === allowedHost.substring(1)) {
-            return true;
-          }
-          // "*.example.com"
-          if (hostname.endsWith(allowedHost)) {
+          // "example.com"  (hostname === allowedHost.substring(1))
+          // "*.example.com"  (hostname.endsWith(allowedHost))
+          if (
+            hostname === allowedHost.substring(1) ||
+            hostname.endsWith(allowedHost)
+          ) {
             return true;
           }
         }
       }
     }
 
-    // allow hostname of listening address
-    if (hostname === this.hostname) {
-      return true;
-    }
-
```

Comment:
Why we remove this?

Reply:
I added this above `hostname === this.hostname;` Line :879

Reply:
:+1: